dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java

```diff
@@ -238,4 +245,29 @@ protected void analyze() {
 		// Rank the graph.
 		execute(e -> new AnalyzeCommand(rootIterator()).execute(e));
 	}
+
+	@Override
+	public TreeNode getTree() {
+		return query(e -> {
+			ResourceIterator<Node> dbroots = e.execute(GET_PHYLO_ROOT).columnAs("n");
+			Node dbroot = dbroots.next();
+
+			return getTree(dbroot, new TreeNode(null));
+		});
+	}
+
+	private TreeNode getTree(Node parent, TreeNode treeparent) {
+		for (Relationship r : IteratorUtil.loop(parent.getRelationships(Direction.OUTGOING,
+										RelTypes.ANCESTOR_OF).iterator())) {
```

Comment:
`getRelationships` returns an `Iterable`, so there's no need to first get the `Iterator` through `iterator()`, and then turn it into an `Iterable` again with `IteratorUtil.loop`.